account_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": 'Accounting with Operating Units',
+    "summary": "Introduces Operating Unit fields in invoices and "
+               "Accounting Entries with clearing account",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "WilldooIT Pty Ltd,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/operating-unit",
+    "category": "Accounting & Finance",
+    "depends": ['account', 'operating_unit', 'analytic_operating_unit'],
+    "license": "LGPL-3",
+    "data": [
+        "security/account_security.xml",
+        "views/account_move_view.xml",
+        "views/account_journal_view.xml",
+        "views/company_view.xml",
+        "views/invoice_view.xml",
+        "views/account_payment_view.xml",
+        "views/account_invoice_report_view.xml",
+    ],
+    'installable': True,
```

Comment:
not necessary
add a cover image for apps-store

Reply:
Not sure about your interpretation: key is not necessary, image is highly recommended :smile:

Reply:
Oh sorry, I misunderstood you. I have updated that and I agree with you in the image issue, but I don't have any "good" image in hand and neither time to do it right now, sorry.